### DISCUSSION TITLE
today's Class

### DISCUSSION THREAD

**Discussion by usamabajwa86** [AammarTufail/python_ka_chilla2023#38] [General]
Aj class ho gi ya nahi?

**Comment by BabarKhalidBaloch** [comment]
i think aaj nae hogi.there is no any update on telegram group

**Reply by AzamBukhari** [reply]
I have email codanics, dropped a message. But No response. Did they informed. What about tomorrow?

**Reply by usamabajwa86** [reply]
don't know about tomorrow,  i did dropped messages yesterday for weekend classes but no update yet
